### DISCUSSION TITLE
Clearing Cache

### DISCUSSION THREAD

**Discussion by MVCreativeServices** [responsively-org/responsively-app#601] [Q&A]
I don't see a way to clear the cache in the app. Is this a feature that could be added? It would be especially nice if each device could have its cache cleared individually, too.

**Comment by jjavierdguezas** [comment]
In the Manage Network menu on the side, there is a Clear Network Cache button, unfortunately it is not per device
![image](https://user-images.githubusercontent.com/13673443/170007146-6da9a821-e277-47ef-8065-e9ef46042396.png)
